golang/Dockerfile

```diff
@@ -13,4 +13,6 @@ RUN go install github.com/codegangsta/gin@latest && \
     go install sigs.k8s.io/controller-tools/cmd/controller-gen@${CONTROLLER_GEN_VERSION} && \
     curl --retry 5 --retry-all-errors -sSfL https://raw.githubusercontent.com/air-verse/air/master/install.sh | sh -s -- -b /usr/bin -d ${AIR_VERSION}
 
+WORKDIR /usr/src/app
```

Comment:
Could we also restore our custom prompt?
https://github.com/okteto/devenv/commit/53bd08ce77a573831de13314706c1c092813188a#diff-bb50b85a394636acf22db527d24325ebe406c3d9d3ce96b974aef72c3c27836aL13